code/SecurityAdmin.php

```diff
@@ -30,20 +31,25 @@
 
 /**
  * Security section of the CMS
+ *
+ * Note: In CMS 5 SecurityAdmin will extend ModelAdmin instead of LeftAndMain
  */
 class SecurityAdmin extends LeftAndMain implements PermissionProvider
 {
 
     private static $url_segment = 'security';
 
+    /**
+     * @deprecated 1.13.0 Will be removed without equivalent functionality to replace it
+     */
     private static $url_rule = '/$Action/$ID/$OtherID';
```

Comment:
```suggestion
    private static $url_rule = '/$Action/$ID/$OtherID';
```
`$url_rule` is very much _not_ deprecated - it comes from `LeftAndMain` and can be configured for SecurityAdmin like it can for any LeftAndMain subclass.

Reply:
Reverted